lib/gds_metrics/path_converter.rb

```diff
@@ -0,0 +1,16 @@
+module GDS
+  module Metrics
+    class PathConverter
+      def self.convert_rails_path_to_route(path)
+        # Convert a path to it's controller and action so we minimise the number of unique timeseries
+        # in the case of IDs/variables in paths. Fallback to path if no route found
+        begin
```

Comment:
Since the scope of the `rescue` is the whole method, you don't need to include the `begin` and `end` - it can be done [like this.](https://stackoverflow.com/questions/1542672/how-does-one-use-rescue-in-ruby-without-the-begin-and-end-block/)

Reply:
thanks, will update :)